lesson02/assignment/index.html

```diff
@@ -1,15 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 02</title>
-    </head>
-    <body>
-        <main class="main">
-            <h1>title</h1>
-        </main>
-    </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>Assignment 02</title>
+</head>
+
+<body>
+  <main class="main static-block">
+    <section class="title">
+      <img src="./img/headshot.jpg" alt="portrait of Paolo Maldini" class="titleimg">
+      <h1>Paolo Maldini</h1>
+      <h2>Front End Developer</h2>
+    </section>
+    <section class="content">
+      <ul>
+        <li>
+          <div class="bold">Company:</div> Web Weavers
+        </li>
+        <li>
+          <div class="bold">Experience:</div> 3 years
+        </li>
+        <li>
+          <div class="bold">School:</div> University of Washington
+        </li>
+        <li>
+          <div class="bold">Major:</div> Marketing
+        </li>
+        <li>
+          <div class="bold">Email:</div> paolo@example.com
```

Comment:
Since we want this bold styling to just change font-weight and not element layout (inline vs block), I'd suggest using a span here rather than a div.